lib/components/Tray.js

```diff
@@ -13,7 +13,8 @@ export default React.createClass({
     closeTimeoutMS: React.PropTypes.number,
     closeOnBlur: React.PropTypes.bool,
     maintainFocus: React.PropTypes.bool,
-    elementToFocus: React.PropTypes.string
+    elementToFocus: React.PropTypes.string,
+    ariaHideElement: React.PropTypes.string
```

Comment:
I still think having these as functions that return an element would be better ;)